packages/frontend/src/types/api.ts

```diff
@@ -12,6 +12,15 @@ export enum SortKeys {
     VoteSum = 'voteSum',
 }
 
+export type FetchReputationHistoryResponse = {
+    score: string
+    report: {
+        reason: string
```

Comment:
`report.reason` means the reason content users write when they make a report. However, the reason column in UI shows why they get scores. You should use `type` in FetchReputationHistoryResponse.

Please check the types in relayer.

https://github.com/social-tw/social-tw-website/blob/12b196ba693037b2ea163fab8b5c3eddab48dbf5/packages/relay/src/types/Reputation.ts#L3-L19